docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Why is this file necessary? So we can build on Windows?

Reply:
Erm... It was generated by sphinx, I didn't really know what it did actually. I will purge it!